leetcode/DIRECTORY.md

```diff
@@ -94,4 +94,5 @@
 | 1184 | [Distance Between Bus Stops](https://leetcode.com/problems/distance-between-bus-stops/)                                         | [C](./src/1184.c) | Easy       |
 | 1189 | [Maximum Number of Balloons](https://leetcode.com/problems/maximum-number-of-balloons/)                                         | [C](./src/1189.c) | Easy       |
 | 1207 | [Unique Number of Occurrences](https://leetcode.com/problems/unique-number-of-occurrences/)                                     | [C](./src/1207.c) | Easy       |
+| 1695 | [Maximum Erasure Value](Maximum Erasure Value)                                                                                  | [C](./src/1695.c) | Medium     |
```

Comment:
Could you please provide the link to this problem instead of repeating the name in the link brackets?

```suggestion
| 1695 | [Maximum Erasure Value](https://leetcode.com/problems/maximum-erasure-value/)
```

Reply:
thanks!